backends/ipvs-as-sink/common.go

```diff
@@ -30,16 +31,56 @@ import (
 	ipsetutil "sigs.k8s.io/kpng/backends/ipvs-as-sink/util"
 )
 
+const (
+	ClusterIPService = "ClusterIP"
+	NodePortService  = "NodePort"
+	LoadBalancerService  = "LoadBalancer"
+)
+
+var protocolIPSetMap = map[string]string{
+	ipsetutil.ProtocolTCP: kubeNodePortSetTCP,
+	ipsetutil.ProtocolUDP: kubeNodePortSetUDP,
+	ipsetutil.ProtocolSCTP: kubeNodePortSetSCTP,
+}
+
+type Operation int32
+
+const (
+	AddService     Operation = 0
+	DeleteService  Operation = 1
+	AddEndPoint    Operation = 2
+	DeleteEndPoint Operation = 3
+)
+
 type endPointInfo struct {
 	endPointIP      string
 	isLocalEndPoint bool
 }
 
+func asDummyIPs(ip string, ipFamily v1.IPFamily) string {
```

Comment:
Two things, 
First why the last ip + "/32" which other value can ipFamily have? Any value other than v4 or v6 should be an error.
Secondly, why the need to prepend a /32 or /128 scope at all? Do you just want to get "vip" address that you can hang on a loopback or use in route.
Given that the name of the function, this is just for my curiosity. No need really to change anything